Cython/Compiler/Nodes.py

```diff
@@ -861,8 +861,9 @@ class CArgDeclNode(Node):
     # annotation     ExprNode or None   Py3 function arg annotation
     # is_self_arg    boolean            Is the "self" arg of an extension type method
     # is_type_arg    boolean            Is the "class" arg of an extension type classmethod
-    # is_kw_only     boolean            Is a keyword-only argument
+    # kw_only        boolean            Is a keyword-only argument
```

Comment:
What's the intention of diverging from the naming scheme here?

Reply:
I just corrected the comment, the variable was already called kw_only before this commit.